### ISSUE TITLE
Can't generate a token

### ISSUE THREAD

**Issue by lpw25** [ocaml/opam-publish#83]
When it tries to generate a token it gets to:
```
<user> 2FA code from 'sms':
```
but I don't get my 2FA message. No idea whether I'm supposed to be doing something to trigger the message or whether its broken.

Also there doesn't seem to be any up-to-date documentation on this tool.

**Comment by ismailbennani** [comment]
I encountered the same problem, I had to switch from SMS 2FA to an app on my phone

**Comment by rjbou** [comment]
Seems that `user` has not been retrieved. @ismailbennani do you have the same message?

**Comment by ismailbennani** [comment]
No, the message I had was `ismailbennani 2FA code from 'sms':` but i didn't receive any sms. I cancelled the command after approx. 10 minutes and tried again, and got the same problem. 
I was using opam-publish 2.0.1 (I upgraded to 2.0.2 after switching to the app 2FA).

**Comment by rjbou** [comment]
We use `ocaml-github` for submitting, [`Token.create`](https://mirage.github.io/ocaml-github/github-unix/Github/Token/index.html#val-create). @avsm any hint ?

**Comment by rjbou** [comment]
Do you still have the issue ?
